File: documentation-examples-scala/src/scala/dev/typr/foundations/docs/sqlserver/UuidType.scala
package dev.typr.foundations.docs.sqlserver
import dev.typr.scalafoundations.*
import dev.typr.scalafoundations.data.*


import java.util.UUID

@SuppressWarnings(Array("unused"))
object UuidType:
  //start
  val uuidType: SqlServerType[UUID] = SqlServerTypes.uniqueidentifier
  //stop

File: documentation-examples-java/src/java/dev/typr/foundations/docs/sqlserver/UuidType.java
package dev.typr.foundations.docs.sqlserver;

import dev.typr.foundations.SqlServerType;
import dev.typr.foundations.SqlServerTypes;
import java.util.UUID;

@SuppressWarnings("unused")
public class UuidType {
  // start
  SqlServerType<UUID> uuidType = SqlServerTypes.uniqueidentifier;
  // stop
}
